source/operations/checklists/software.rst

```diff
@@ -59,7 +59,7 @@ MinIO Pre-requisites
      - :ref:`Prometheus <minio-metrics-collect-using-prometheus>` or a Prometheus-compatible setup for monitoring and metrics
 
    * - :octicon:`circle`
-     - :ref:`Grafana configured <minio-grafana>` for dashboards 
+     - (optional) `Grafana <https://grafana.com/>`__ for custom dashboards 
```

Comment:
I guess say this? For metrics v3 you are on your own for a Grafana dashboard.

Could also just remove reference to Grafana entirely.